README.md

```diff
@@ -113,7 +133,8 @@ end
 Your answer:
 ```text
 
-Replace this with your answer
+The Context scenarios are adding more detailed tests to the broader Describe test.  Since there is a restriction of 3 bedrooms, it makes sense to highlight adding a "third tenant".
+
```

Comment:
true, sort of! the important thing to note is that ```describe``` tells you what unit is being tested, while context tells you the conditions that are tested